test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1457,7 +1465,7 @@ def load_default_config(self, mom_checks=True):
              'content-encoding': 'default',
              'input-file': self.config_file}
         self.server.manager(MGR_CMD_IMPORT, HOOK, a, self.hook_name)
-        if not mom_checks:
+        if not mom_checks or self.moms_list[0].is_cpuset_mom():
```

Comment:
Don't we want to verify the configuration change has been made on a cpuset mom?  Did you mean to check mom.revert_to_default instead?

Reply:
True, I should be checking mom.revert_to_default instead. I'll update.

Reply:
It's not necessary under cpuset mom to verify the default hook config file was loaded in the local mom, since this pbs_cgroups hook testsuite uses its own hook config file at each test case.